### PR TITLE
Scheduled events

### PR DESCRIPTION
**Pull request by MGathier**

### CODE REVIEW

File: axonserver/src/test/java/io/axoniq/axonserver/taskscheduler/LocalTaskManagerTest.java [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+                                    Duration.ofMinutes(11));
+        assertEquals(1, tasks.size());
+        assertEquals(1, scheduler.tasks());
+        scheduler.timeElapses(5, TimeUnit.MINUTES);
+        assertEquals(1, tasks.size());
+        assertEquals(2, scheduler.tasks());
+
+        scheduler.timeElapses(6, TimeUnit.MINUTES);
+        assertEquals(0, tasks.size());
+        assertEquals(1, scheduler.tasks());
```
Comment (smcvb) [inline axonserver/src/test/java/io/axoniq/axonserver/taskscheduler/LocalTaskManagerTest.java]: Slightly at a lose here. The remaining task on the "Scheduler" is the task to retrieve potential tasks, right?

File: axonserver/src/main/resources/db/migration/V4_0__tasks_update.sql [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    context        varchar(200) not null,
+    data           blob,
+    type           varchar(200),
+    status         INTEGER      not null,
+    retry_interval BIGINT,
+    task_executor  varchar(200) not null,
+    timestamp      BIGINT       not null
+);
+
+alter table task
```
[the remark this answers is not in the document]
Comment (MGathier) [reply axonserver/src/main/resources/db/migration/V4_0__tasks_update.sql:13]: Task table already exists in EE. create if missing adds the table  in SE with same definition as in EE.

File: axonserver/src/main/java/io/axoniq/axonserver/taskscheduler/JacksonTaskPayloadSerializer.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+/**
+ * Serializer to serialize task payloads in JSON format.
+ *
+ * @author Marc Gathier
+ * @since 4.3
+ */
+@Component
+public class JacksonTaskPayloadSerializer implements TaskPayloadSerializer {
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
```
[the remark this answers is not in the document]
Comment (MGathier) [reply axonserver/src/main/java/io/axoniq/axonserver/taskscheduler/JacksonTaskPayloadSerializer.java:27]: Don't think so, as we are not creating a framework here.

[the remark this answers is not in the document]
Comment (smcvb) [reply axonserver/src/main/java/io/axoniq/axonserver/taskscheduler/JacksonTaskPayloadSerializer.java:27]: Fair argument, had a different mindset here.

File: axonserver/src/main/resources/db/migration/V4_0__tasks_update.sql (hunk 2 above)
[the remark this answers is not in the document]
Comment (smcvb) [reply axonserver/src/main/resources/db/migration/V4_0__tasks_update.sql:13]: Awesome, thanks for clarifying that!

**Pull request merged**